delivery_transsmart/wizards/stock_transfer_details.py

```diff
@@ -0,0 +1,16 @@
+# -*- coding: utf-8 -*-
+# Copyright 2018 Therp BV <https://therp.nl>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openerp import api, models
+
+
+class StockTransferDetails(models.TransientModel):
+    _inherit = 'stock.transfer_details'
+
+    @api.multi
+    def do_detailed_transfer(self):
+        self.ensure_one()
+        if self.picking_id._send_to_transsmart():
+            # TODO how will partial moves be taken care off?
```

Comment:
@lfreeke Maybe this is a question for you? When a picking is to be sent on transsmart, should the user be prohibited or making it a partial or should I create a document for every part of the picking?